### DISCUSSION TITLE
Unable to come to consensus about closing feerate

### DISCUSSION THREAD

**Discussion by ConorOkus** [lightningdevkit/rust-lightning#2826] [Help] [answered]
I get the following error when trying to initiate a coop close (regtest).

`Unable to come to consensus about closing feerate, remote wants something (8145 sat) higher than our max fee (1182 sat)`

This causes a force close and I never seem to get a `SpendableOutputs` event after waiting the required time for confirmations. Peer is LND v0.17.3-beta

**Comment by TheBlueMatt** [comment, marked as answer]
> This causes a force close

Sadly there's only so much we can do about this for now. This is a pretty common issue for various reasons (often Bitcoin Core providing fee estimates without having enough history to provide robust fee estimates or just nodes using drastically different fee estimators). We're hoping the "v3 coops close negoation" protocol fixes it by removing this class of errors.

> This causes a force close and I never seem to get a SpendableOutputs event after waiting the required time for confirmations.

How long did you wait? Note that because its a force-close you have to wait the to-self-delay which is potentially a day or two.

**Reply by ConorOkus** [reply]
Thanks, after playing around with fees the coop close case worked fine. Had to wait for the 144 blocks for the force close 👍🏿
